lib/geordi/commands/checkout.rb

```diff
@@ -0,0 +1,16 @@
+desc 'checkout', 'Checkout a feature branch using a story title from Pivotal Tracker'
+long_desc <<-LONGDESC
+Example: `geordi checkout`
+
+On the first execution we ask for your Pivotal Tracker API token. It will be
+stored in `~/.config/geordi/global.yml`.
+LONGDESC
+
+def checkout(*git_args)
```

Comment:
Your command does not take any arguments, so you should not take them here either.